### ISSUE TITLE
Avoid dumping 0 instead of undefined values for HVDC power values when applicable

### ISSUE THREAD

**Issue by rosiereflo** [dynawo/dynawo#3049]

**Comment by gautierbureau** [comment]
Do you mean this https://github.com/dynawo/dynawo/pull/1957/files#diff-611cbbb4dad5100bc98cd32eee29540478ed9ac0cf9f1fb93841081ceb25ff02 ?

**Comment by rosiereflo** [comment]
yes but only for hvdc and strongly validated as it creates false power values in platforms

**Comment by barondim** [comment]
- [ ] To be merged with https://github.com/dynawo/dynaflow-launcher/pull/723
